Nodejs/Product/Nodejs/NpmUI/NpmWorker.cs

```diff
@@ -106,74 +106,135 @@ private void Run()
 
             TelemetryHelper.LogSearchNpm();
```

Comment:
We don't care which version of search we used?

Reply:
Nope